aspnetcore/getting-started-1.1.md

```diff
@@ -23,9 +23,6 @@ uid: getting-started-1.1
 
 1. Install [.NET Core](https://microsoft.com/net/core).
 
-   If you're on Windows, select the **Command line / other** environment. 
```

Comment:
@danroth27  Can we change this to  
If you're on Windows and you don't want to install Visual Studio, select the **Command line / other** environment.

Installing VS is significant/unnecessary friction for someone who just wants a quick test run.